src/containers/org/OrgRouter.js

```diff
@@ -303,6 +321,7 @@ const OrgRouter = () => {
 
     return (
       <Switch>
+        <Route exact path={Routes.ENTITY_DETAILS} render={renderDataSetDataDetailsContainer} />
```

Comment:
I'm not quite sure where the bug is, but I can't view the non-modal version of `EntityDataContainer`. If I click the "Get Link" button to copy the url and then paste and navigate to it, then `EntityDataContainer` doesn't get rendered.